packages/backend/src/vitest.setup.integration.ts

```diff
@@ -152,6 +154,30 @@ export const setupIntegrationTest =
             abilityRules: [],
         };
 
+        const testUserSessionAccount: SessionAccount = {
```

Comment:
`findExplore/getExplore` expects new type `Account` (which can be `SessionAccount`)